nexus/src/authn/external/spoof.rs

```diff
@@ -59,34 +97,63 @@ where
 }
 
 fn authn_spoof(raw_value: Option<&http::HeaderValue>) -> SchemeResult {
-    if let Some(raw_value) = raw_value {
-        match raw_value.to_str().context("parsing header value as UTF-8") {
-            Err(source) => SchemeResult::Failed(Reason::BadFormat { source }),
-            Ok(str_value) if str_value == SPOOF_RESERVED_BAD_ACTOR => {
-                SchemeResult::Failed(Reason::UnknownActor {
-                    actor: str_value.to_owned(),
-                })
-            }
-            Ok(str_value) if str_value == SPOOF_RESERVED_BAD_CREDS => {
-                SchemeResult::Failed(Reason::BadCredentials {
-                    actor: *SPOOF_RESERVED_BAD_CREDS_ACTOR,
-                    source: anyhow!("do not sell to the people on this list"),
-                })
-            }
-            Ok(str_value) => match Uuid::parse_str(str_value)
-                .context("parsing header value as UUID")
-            {
-                Ok(id) => {
-                    SchemeResult::Authenticated(Details { actor: Actor(id) })
-                }
-                Err(source) => {
-                    SchemeResult::Failed(Reason::BadFormat { source })
-                }
-            },
-        }
-    } else {
-        SchemeResult::NotRequested
+    let decode_result =
+        <Authorization<Bearer> as Header>::decode(&mut raw_value.into_iter());
```

Comment:
seems to work without the `as Header`

```suggestion
        <Authorization<Bearer>>::decode(&mut raw_value.into_iter());
```

Reply:
Fixed.